src/Core/src/Handlers/Button/ButtonHandler.iOS.cs

```diff
@@ -45,14 +45,12 @@ protected override void SetupDefaults(UIButton nativeView)
 			base.SetupDefaults(nativeView);
 		}
 
-		public static void MapBackgroundColor(ButtonHandler handler, IButton button)
-		{
-			handler.NativeView?.UpdateBackgroundColor(button);
-		}
-
 		public static void MapText(ButtonHandler handler, IButton button)
 		{
 			handler.NativeView?.UpdateText(button);
+
+			// Any text update requires that we update any attributed string formatting
+			MapFormatting(handler, button);
```

Comment:
So many things need to update the formatting after text changes now. Ordering was a lie.